standalone/src/test/java/io/pravega/local/AuthEnabledInProcPravegaClusterTest.java

```diff
@@ -52,105 +38,61 @@ public void setUp() throws Exception {
-        log.info("Done writing event [{}]", writeEvent2);
+    @Override
+    String streamName() {
+        return "AuthTestStream";
+    }
 
-        // Now, read the events from the stream.
+    @Override
+    String eventMessage() {
+        return "Test message on the plaintext channel with auth credentials";
+    }
 
-        String readerGroup = UUID.randomUUID().toString().replace("-", "");
-        ReaderGroupConfig readerGroupConfig = ReaderGroupConfig.builder()
-                .stream(Stream.of(scope, streamName))
-                .disableAutomaticCheckpoints()
+    @Override
+    ClientConfig prepareValidClientConfig() {
+        return ClientConfig.builder()
+                .controllerURI(URI.create(localPravega.getInProcPravegaCluster().getControllerURI()))
+                .credentials(new DefaultCredentials("1111_aaaa", "admin"))
                 .build();
-
-        @Cleanup
-        ReaderGroupManager readerGroupManager = ReaderGroupManager.withScope(scope, clientConfig);
-        readerGroupManager.createReaderGroup(readerGroup, readerGroupConfig);
-
-        @Cleanup
-        EventStreamReader<String> reader = clientFactory.createReader(
-                "readerId", readerGroup,
-                new JavaSerializer<String>(), ReaderConfig.builder().build());
-
-        // Keeping the read timeout large so that there is ample time for reading the event even in
-        // case of abnormal delays in test environments.
-        String readEvent1 = reader.readNextEvent(10000).getEvent();
-        log.info("Done reading event [{}]", readEvent1);
-
-        String readEvent2 = reader.readNextEvent(10000).getEvent();
-        log.info("Done reading event [{}]", readEvent2);
-
-        assertEquals(writeEvent1, readEvent1);
-        assertEquals(writeEvent2, readEvent2);
     }
 
-    @Test
-    public void testCreateStreamFailsWhenCredentialsAreInvalid() {
-        Assert.assertNotNull("Pravega not initialized", localPravega);
-        String scope = "Scope";
-
-        ClientConfig clientConfig = ClientConfig.builder()
+    /**
+     * This test verifies that create stream fails when the client config is invalid.
+     *
+     * Note: The timeout being used for the test is kept rather large so that there is ample time for the expected
+     * exception to be raised even in case of abnormal delays in test environments.
+     */
+    @Test(timeout = 300000)
+    public void testCreateStreamFailsWithInvalidClientConfig() {
+       ClientConfig clientConfig = ClientConfig.builder()
                 .credentials(new DefaultCredentials("", ""))
                 .controllerURI(URI.create(localPravega.getInProcPravegaCluster().getControllerURI()))
                 .build();
 
         @Cleanup
         StreamManager streamManager = StreamManager.create(clientConfig);
 
-        AssertExtensions.assertThrows(RuntimeException.class,
-                () -> streamManager.createScope(scope));
+        try {
```

Comment:
Also, please use assertExtensions.assertThrows with the predicate check for thrown exception

Reply:
Thanks for catching the bug in the test. Have fixed it now in the latest commit. The assertion code is also looking simpler with that change.